scripts/rust_ci_helper.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+import json
+import argparse
+import subprocess
+import os
+
+
+def run(command):
+    assert isinstance(command, str)
+    print(f"Running: {command}")
+    exit_code = subprocess.check_call(command, shell=True, text=True)
+    assert exit_code == 0
+
+
+def run_with_stdout(command, print_cmd=True):
+    assert isinstance(command, str)
+    if print_cmd:
```

Comment:
I would say just always print it. you are writing some CI middlewere here. visibility is important.